cmd/standalone/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.7
+EXPOSE 5775 6831 6832 14267 3000 3001 8080 8081 8082 8083
+
+ADD . /go/src/github.com/uber/jaeger
```

Comment:
the crossdock tests do the build outside of container and only copy the binary. Why would we do this differently?